tools/src/commands/PublishProjectTemplates.ts

```diff
@@ -90,16 +115,6 @@ async function action(options) {
       },
     ]);
 
-    // Obtain the tag for the template.
-    const { tag } = await inquirer.prompt<{ tag: string }>([
-      {
-        type: 'input',
-        name: 'tag',
-        message: `How to tag ${chalk.green(template.name)}@${chalk.red(newVersion)}?`,
-        default: semver.prerelease(newVersion) ? 'next' : `sdk-${semver.major(options.sdkVersion)}`,
```

Comment:
There was already a check for `semver.prerelease(newVersion)` here

Reply:
ah gotcha. interesting. might as well leave it in place then